sklearn/_config.py

```diff
@@ -61,8 +61,8 @@ def set_config(assume_finite=None, working_memory=None,
         .. versionadded:: 0.21
 
     display : {'text', 'diagram'}, optional
-        If 'diagram', estimators will be displayed as text in a jupyter lab
-        of notebook context. If 'text', estimators will be displayed as
+        If 'diagram', estimators will be displayed as a diagram in a Jupyter
```

Comment:
The `display` keyword in `config_context` should be updated as well.

Reply:
Done!